src/main/java/ru/sbt/mipt/oop/SmartHome.java

```diff
@@ -14,11 +16,48 @@ public SmartHome(Collection<Room> rooms) {
         this.rooms = rooms;
     }
 
+    public void processDoorEvent(SensorEvent event) {
```

Comment:
https://github.com/iagoshkov/smart-home-2020/blob/master/STUDENT_GUIDE.md#%D0%B4%D0%B5%D0%BB%D0%B5%D0%B3%D0%B8%D1%80%D1%83%D0%B9%D1%82%D0%B5-%D0%BE%D0%B1%D1%80%D0%B0%D0%B1%D0%BE%D1%82%D0%BA%D1%83-%D1%81%D0%BE%D0%B1%D1%8B%D1%82%D0%B8%D0%B9-%D0%BE%D1%82%D0%B4%D0%B5%D0%BB%D1%8C%D0%BD%D1%8B%D0%BC-%D0%BA%D0%BB%D0%B0%D1%81%D1%81%D0%B0%D0%BC